go.sum

```diff
@@ -780,6 +763,8 @@ github.com/prometheus/procfs v0.0.0-20181005140218-185b4288413d/go.mod h1:c3At6R
 github.com/prometheus/procfs v0.0.2/go.mod h1:TjEm7ze935MbeOT/UhFTIMYKhuLP4wbCsTZCD3I8kEA=
 github.com/prometheus/procfs v0.1.3/go.mod h1:lV6e/gmhEcM9IjHGsFOCxxuZ+z1YqCvr4OA4YeYWdaU=
 github.com/prometheus/procfs v0.6.0/go.mod h1:cz+aTbrPOrUb4q7XlbU9ygM+/jj0fzG6c1xBZuNvfVA=
+github.com/pvormste/websocket v1.8.8 h1:W/XXNVEsK7696u919b+fp1cHZxb6EUr0WhQcYXV3rTE=
+github.com/pvormste/websocket v1.8.8/go.mod h1:ygCxO/vU7HwVfV36RpJqZK2biPnjNx7OJP45DrbCcvk=
```

Comment:
@pvormste release code shouldn't live in personal repos, is it fine adding a dependency onto that?

cc @buger